docs/_includes/attr-use.adoc

```diff
@@ -32,22 +30,13 @@ and
 
  :!sectnums:
 
-means unset the `sectnums` attribute and therefore “do not number” the sections.
-
-The following sections will show you how to use attributes on your whole document, individual blocks, and inline elements.
-
-////
-
-NOTE: To learn about command line attributes and how to use them, proceed to <<processing-a-document>>.
+mean unset the `sectnums` attribute and therefore “do not number” the sections.
 
-Attributes can be set via the command line, in the document header, and on individual block and inline elements.
+An attribute *reference* is the name of the attribute enclosed in curly brackets:
```

Comment:
I think we are missing that an attribute reference is an inline construct. In other words, where can it be used.